tests/job_runner_tsan_suppressions.txt

```diff
@@ -1,3 +1,5 @@
 # Boost lockfree queue triggers the data race warning, which is a known issue.
 # https://github.com/boostorg/lockfree/issues/78
 race:boost/lockfree/detail/tagged_ptr_ptrcompression.hpp
+race:boost/lockfree/detail/copy_payload.hpp
```

Comment:
Are they also known issues?

Reply:
It should be the same/similar problem here. It is within the push/pop functions too, and the stacktrace does not include our code. I have not digged into it though.